src/doc/man/includes/options-locked.md

```diff
@@ -18,8 +18,10 @@ proceed without the network if possible.
 Beware that this may result in different dependency resolution than online
 mode. Cargo will restrict itself to crates that are downloaded locally, even
 if there might be a newer version as indicated in the local copy of the index.
+{{#if (ne no-fetch-in-offline true)}}
```

Comment:
Would this work?

```suggestion
{{#if (ne command "fetch")}}
```

Reply:
Yes, it works.   I should have tried this before, but it didn't work.